stylesheets/_component.modals.scss

```diff
@@ -3,159 +3,161 @@
 // --------------------------------------------------
 
 // .modal--open      - body class for killing the scroll
-// .modal           - container to scroll within
+// .modal            - container to scroll within
 // .modal__dialog    - positioning shell for the actual modal
 // .modal__content   - actual modal w/ bg and corners
+// .modal--fs        - modifier for a full screen modal
```

Comment:
Is this an alias? isn't your class for full screen modals `.modal--full-screen`?